index.html

```diff
@@ -185,5 +185,15 @@ <h3 class="languages-title">Languages</h3>
             <img src="images/separator.png" class="separator" alt="separator">
         </div>
     </section>
+    <section class="contact">
+        <h1 class="contact-title">Contact me</h1>
+        <p class="contact-subtitle">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
+        <form class="form-section" method="post" action="https://formspree.io/f/xdojkarj">
+            <input type="text" placeholder="Enter your name" required maxlength="30"/>
+            <input type="email" name="email" placeholder="Enter your email" required />
+            <textarea placeholder="Write your message here" maxlength="500"></textarea>
```

Comment:
- [ ] Kindly, add `required` attribute for text area validation, All input fields must have the `required` attribute to activate the validation on submit.